redwood-treehouse-host/src/androidMain/kotlin/app/cash/redwood/treehouse/treehouseAppFactoryAndroid.kt

```diff
@@ -34,12 +36,13 @@ public fun TreehouseAppFactory(
   context: Context,
   httpClient: OkHttpClient,
   manifestVerifier: ManifestVerifier,
-  eventListener: EventListener = EventListener(),
+  eventListener: EventListener = defaultEventListener,
   embeddedDir: Path? = null,
   embeddedFileSystem: FileSystem? = null,
   cacheName: String = "zipline",
   cacheMaxSizeInBytes: Long = 50L * 1024L * 1024L,
   concurrentDownloads: Int = 8,
+  stateStore: StateStore = MemoryStateStore(),
```

Comment:
during testing: the process would get killed and the in memory state store would be empty when navigating back

Reply:
Makes sense. This thing would probably work great as long as 'kill activities' isn't enabled